sdk/servicebus/azure-messaging-servicebus-stress/README.md

```diff
@@ -0,0 +1,74 @@
+# Azure Service Bus Stress test client library for Java
+
+Represents stress tests for Service Bus client.
+
+## Getting started
+
+The stress tests for service bus client is developed from [azure-sdk-chaos][azure_sdk_chaos]. 
+
+Follow the same startup steps in [Event Hubs Stress Test - Getting started][event_hubs_stress_test_start].
```

Comment:
Becasue the SB start up steps is totally same as EH, so I link this to Event Hubs Stress Test. 
Do you think it is good to link EH or copy the content to SB?

Reply:
I copied the content to SB instead of using the EH link. It's more straightforward and the user doesn't need to open a new link to get started. The disadvantage is that we need to maintain two README files.